tests/test_security_list.py

```diff
@@ -24,7 +24,10 @@
 
 class RestrictedAlgoWithCheck(TradingAlgorithm):
     def initialize(self, symbol):
-        self.rl = SecurityListSet(self.get_datetime, self.asset_finder)
+        self.rl = SecurityListSet(
+            self.get_datetime,
+            self.asset_finder,
+            load_from_directory('leveraged_etf_list'))
```

Comment:
Is there a reason we're loading this in every test rather than just relying on `LEVERAGED_ETFS`?